core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -146,10 +146,15 @@ private void registerAggregators() {
         MetricsSystem.InstanceType.CLIENT, ClientMetrics.BYTES_READ_LOCAL_THROUGHPUT));
 
     // multi-value aggregators
-    addAggregator(new SingleTagValueAggregator(MetricsSystem.InstanceType.WORKER,
-        WorkerMetrics.BYTES_READ_UFS, WorkerMetrics.TAG_UFS));
-    addAggregator(new SingleTagValueAggregator(MetricsSystem.InstanceType.WORKER,
-        WorkerMetrics.BYTES_WRITTEN_UFS, WorkerMetrics.TAG_UFS));
+    addAggregator(new SingleTagValueAggregator(WorkerMetrics.BYTES_READ_UFS,
+        MetricsSystem.InstanceType.WORKER, WorkerMetrics.BYTES_READ_UFS, WorkerMetrics.TAG_UFS));
+    addAggregator(new SingleTagValueAggregator(WorkerMetrics.BYTES_WRITTEN_UFS,
+        MetricsSystem.InstanceType.WORKER, WorkerMetrics.BYTES_WRITTEN_UFS, WorkerMetrics.TAG_UFS));
+    for (WorkerMetrics.UfsOps ufsOp : WorkerMetrics.UfsOps.values()) {
```

Comment:
Could we add these on the fly as we get them, then we would not have to keep track of a master list.

Reply:
hmm how to do it?

Reply:
I was thinking of doing something like prefixing all the ops with `UfsOp` and then if a metric comes in with the prefix we would create an aggregator for it if it has not been created. But this approach is also not clean, and I'm okay with the enum if you think that is preferable.

Reply:
Right. I think a central place for registration offers better modularity and code cleanness. So I feel it's worth maintaining an extra enum data structure